source/includes/template_config/_00-describe.md.erb

```diff
@@ -1,6 +1,7 @@
 <%=
-describe_object 'The template config object', since: '16.10.0' do
+describe_object 'The template config object', since: '19.9.0' do
```

Comment:
This should remain as `16.10.0` since this API was introduced in that release.

Reply:
Duh. Got it